ppanggolin/workflow/all.py

```diff
@@ -363,6 +364,10 @@ def add_workflow_args(parser: argparse.ArgumentParser):
     optional.add_argument("--identity", required=False, type=restricted_float, default=0.8,
                           help="Minimal identity percent for two proteins to be in the same cluster")
 
+    optional.add_argument("--infer_singletons", required=False, action="store_true",
```

Comment:
A part of the help is missing, so we don't really understand this option... 
An alternative: 

Use this option together with --clusters. If a gene is not present in the provided clustering result file, it will be assigned to its own unique cluster as a singleton.